front/lib/models/assistant/actions/retrieval.ts

```diff
@@ -326,11 +326,11 @@ RetrievalDocument.init(
 );
 
 AgentRetrievalAction.hasMany(RetrievalDocument, {
```

Comment:
I guess we need to update the type line 273?

Reply:
👍🏼